libraries/stdlib/test/numbers/MathTest.kt

```diff
@@ -121,6 +121,30 @@ class DoubleMathTest {
         }
     }
 
+    @Test fun cubeRoots() {
+        val testingPairs = mapOf(
+            Double.NaN to Double.NaN,
+            Double.POSITIVE_INFINITY to Double.POSITIVE_INFINITY,
+            Double.NEGATIVE_INFINITY to Double.NEGATIVE_INFINITY,
+            Double.fromBits(0x0010000000000000) to 2.812644285236262E-103, // smallest normal value
+            Double.fromBits(0x1L) to 1.7031839360032603E-108, // smallest value (denormal)
+            Double.MAX_VALUE to 5.643803094122362E102,
+            0.0 to 0.0,
+            0.9 to 0.9654893846056297,
+            9.9 to 2.1472291690189413,
+            27.0 to 3.0,
+            399.1289 to 7.362710510208026,
+            8123.452 to 20.102351976782558,
+            21717.639 to 27.9,
+            392890.22 to 73.24147345684439
+        )
+
+        for ((x, result) in testingPairs) {
+            assertEquals(result, cbrt(x), 1e-118)
```

Comment:
I don't think that the absolute tolerance is suitable here. For example, it fails in test for `kotlin-stdlib-js` with "Expected <0.9654893846056297> with absolute tolerance <1e-118>, actual <0.9654893846056298>".

Reply:
I was afraid of that once the tests would run on a different machine. I'll change to a safer tolerance.

Reply:
I like that idea a lot, I'll give it a try. Thanks for the suggestion!